test.py

```diff
@@ -1,12 +1,12 @@
 import asyncio
 from hachiko.hachiko import AIOWatchdog
 
-@asyncio.coroutine
-def watch_fs():
-    watch = AIOWatchdog('/Users/jbiesnecker/temp/forks/test')
+
+async def watch_fs():
+    watch = AIOWatchdog('/home/genevieve/Documents/Projects/watchdir/code/data/watch_dir/')
```

Comment:
Ideally this wouldn't be committed, and we'd switch to something a bit more proper for the tests. This is fine for now, though.

Reply:
Aw crap!